elide-standalone/src/test/java/example/ElideStandaloneTest.java

```diff
@@ -159,6 +162,20 @@ public void testJsonAPIPost() {
             .then()
             .statusCode(HttpStatus.SC_CREATED)
             .extract().body().asString();
+
+        // Test the Dynamic Generated Analytical Model is accessible
+        given()
+            .when()
+            .get("/api/v1/postView")
+            .then()
```

Comment:
You can combine this test with the one below in a single request (test the body and the status code).